cpp/ps0007/p0007.soln.cpp

```diff
@@ -30,10 +30,13 @@ double area(char shape, double dimensions[]) {
     }
 }
 int main() {
+    // prompting user for the shape 1.5/1.5
     char shape;
     cout << "Enter shape (C for Circle, S for Square, R for Rectangle): ";
     cin >> shape;
     double dimensions[2] = {0, 0};

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input